app/Http/Controllers/Api/HetznerController.php

```
@@ -0,0 +1,651 @@
+
+            // Create server in Coolify database
+            $server = Server::create([
+                'name' => $request->name,
+                'ip' => $ipAddress,
+                'user' => 'root',
+                'port' => 22,
+                'team_id' => $teamId,
+                'private_key_id' => $privateKey->id,
+                'cloud_provider_token_id' => $token->id,
+                'hetzner_server_id' => $hetznerServer['id'],
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method 'createServer'. (undefined)

(StaticAccess)